notifications_utils/jinja_templates/email/_custom_logo_no_background_colour.jinja2

```diff
@@ -9,7 +9,7 @@
         <tr>
           <td style="padding: 0 10px 0 {% if brand_colour %} 8px; border-left: solid 2px {{ brand_colour }}{% else %} 10px;{% endif %}">
             <img src="{{ brand_logo }}" style="display: block; border: 0" height="{% if brand_text -%} 27 {%- else -%} 108 {%- endif %}"
-                        alt="{% if brand_text %} {% else -%}{{ brand_name }}{%- endif %}" />
+                        alt="{% if alt_text_en %}{{ alt_text_en }} / {{ alt_text_fr }}{% else -%}{{ brand_name}}{%- endif %}" />
```

Comment:
should we be passing both in, instead of just the english? the api code makes sure each brand has both an alt_text_en and an alt_text_fr - both the fields cannot be empty. Do we need the if statement?